mappings/net/minecraft/entity/player/InventoryPlayer.mapping

```diff
@@ -19,8 +19,21 @@ CLASS aal net/minecraft/entity/player/InventoryPlayer
 	METHOD a setInvStack (ILaes;)V
 		ARG 0 slot
 		ARG 1 stack
+	METHOD a clone (Laal;)V
 	METHOD a canPlayerUseInv (Laam;)Z
 		ARG 0 player
+	METHOD a clear (Laeq;IILdt;)I
+		ARG 0 item
+		ARG 1 meta
+		ARG 2 amount
+		ARG 3 tag
+	METHOD a addPickBlock (Laes;)V
+		ARG 0 stack
+	METHOD a canStackAddMore (Laes;Laes;)Z
+		ARG 0 existingStack
+		ARG 1 stackToAdd
+	METHOD a getCurrentMiningStrength (Lasm;)F
```

Comment:
I suppose that makes sense, as `getMiningStrength` could imply it would only change on a call to a setter.